src/sentry/tasks/post_process.py

```diff
@@ -210,6 +211,12 @@ def post_process_group(
                 ):
                     safe_execute(callback, event, futures)
 
+            if features.has("organizations:workflow-owners", project=event.project):
```

Comment:
You won't be able to use flagr for this feature flag, as this is in the event ingestion flow.

Reply:
Thanks for the heads up. Didn't realize flagr was much slower. Creating a new flag just to use in ingestion, so that I can use this one to add owners for other teams within sentry eventually.

Reply:
Ended up sticking with one flag and using a Workflow team handler. No more flagr 😢